### PR TITLE
Add pipcook

### PR DESCRIPTION
**Pull request by yorkie**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

⬆⬆⬆⬆⬆⬆⬆⬆⬆⬆

The pipcook is a front-end algorithm framework to create a ML pipeline based on tfjs-node in JavaScript, and it's included in tfjs gallery: https://github.com/tensorflow/tfjs/blob/master/GALLERY.md

GitHub: https://github.com/alibaba/pipcook

### CODE REVIEW

**Comment by yorkie** [comment]
Ping @sindresorhus :)

**Comment by yorkie** [comment]
Hi @sindresorhus May I ask you help me review this again, is there another reason that you are not going to merge this?

File: readme.md [hunk 1]
```diff
@@ -125,6 +125,7 @@
 - [NodeOS](https://github.com/NodeOS/NodeOS) - The first operating system powered by npm.
 - [YodaOS](https://github.com/yodaos-project/yodaos) - AI operating system.
 - [Brain.js](https://github.com/BrainJS/brain.js) - Machine-learning framework.
+- [Pipcook](https://github.com/alibaba/pipcook) - Front-end algorithm framework to create a ML pipeline based on tfjs-node.
```
Comment (sindresorhus) [inline readme.md:128]: tfjs-node is deprecated: https://github.com/tensorflow/tfjs-node

`ML` => machine learning

**Comment by sindresorhus** [comment]
I review in batches once in a while.

**Pull request merged**